src/components/NotificationBannerModule.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <header id="CVE-Notification-Banner" class="is-info" role="alert" v-if="shouldDisplay == 'true'">
+    <div class="notification is-info" style="padding: 10px;">
+      <button class="delete" style="margin:auto; float:right;"></button>
+      <div class="is-flex" style="justify-content: left; margin:auto;">
+          <p id="alertIcon" class="is-hidden"></p>
+          <font-awesome-icon style="flex: 0 0 40px; margin-top:3px" size="1x"  icon="info-circle" role="alert"
+            aria-labelledby="alertIcon" aria-hidden="false"/>
+          <div>
+            <span v-for="(section, index) in notificationContents" :key="index">
+              <span v-if="section.contentType == 'paragraph'" v-html="$sanitize(section.content) + ' '"></span>
+              <span class="has-text-weight-bold" v-if="section.contentType == 'bold'">{{section.content}} </span>
+              <a v-if="section.contentType == 'externalLink'" :href="section.link">{{section.linkText}}
+                <span class="icon cve-icon-xxs">
+                  <p :id="'bannerExtenalLink' + index" class="is-hidden">external link</p>
+                  <font-awesome-icon icon="external-link-alt" :aria-labelledby="'bannerExtenalLink' + index"/>
+                </span>
+                {{ ' ' }}
+              </a>
+              <span v-if="section.contentType == 'internalLink'">
+                <router-link :to="section.link">{{section.linkText}} </router-link>
+              </span>
+            </span>
+          </div>
+      </div>
+    </div>
+  </header>
+</template>
+
+<script>
+import NotificationData from '@/assets/data/NotificationBanner.json';
+
+export default {
+  data() {
+    return {
+      shouldDisplay: NotificationData.shouldDisplay,
+      notificationContents: NotificationData.notificationContents,
+    };
+  },
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  (document.querySelectorAll('.notification .delete') || []).forEach(($delete) => {
+    const $notification = $delete.parentNode;
+
+    $delete.addEventListener('click', () => {
+      $notification.parentNode.removeChild($notification);
+    });
+  });
+});
```

Comment:
This is not a Vue convention...please remove this block of code and refer to feedback on line 4 on `v-on:click` attribute.